e2e/scaffolding/P256.ts

```diff
@@ -0,0 +1,76 @@
+import { SubmittableExtrinsic } from '@polkadot/api/types';
+import { base64UrlToUint8Array } from './helpers';
+import { secp256r1 } from '@noble/curves/p256';
+import { ISubmittableResult } from '@polkadot/types/types';
+import { u8aWrapBytes } from '@polkadot/util';
+import { ExtrinsicHelper } from './extrinsicHelpers';
+import { sha256 } from '@noble/hashes/sha256';
+
+export function createPassKeyAndSignAccount(accountPKey: Uint8Array) {
+  const passKeyPrivateKey = secp256r1.utils.randomPrivateKey();
+  const passKeyPublicKey = secp256r1.getPublicKey(passKeyPrivateKey, true);
+  const passkeySignature = secp256r1.sign(u8aWrapBytes(accountPKey), passKeyPrivateKey).toDERRawBytes();
+  return { passKeyPrivateKey, passKeyPublicKey, passkeySignature };
+}
+
+export async function createPassKeyCall(
+  accountPKey: Uint8Array,
+  nonce: number,
+  accountSignature: Uint8Array,
+  call: SubmittableExtrinsic<'rxjs', ISubmittableResult>
+) {
+  const ext_call_type = ExtrinsicHelper.api.registry.createType('Call', call);
+  const passkeyCall = {
+    accountId: accountPKey,
+    accountNonce: nonce,
+    accountOwnershipProof: {
+      Sr25519: accountSignature,
+    },
+    call: ext_call_type,
+  };
+
+  return passkeyCall;
+}
+
+export async function createPasskeyPayload(
+  passKeyPrivateKey: Uint8Array,
+  passKeyPublicKey: Uint8Array,
+  passkeyCallPayload: any = {},
+  bad: boolean = false
```

Comment:
This was not the easiest to understand when reading through the tests, since you are just passing a boolean to get an invalid payload back.  It would be clearer to split these up or else pass named option(s)

Reply:
Gave is a meaningful name